prow/config.yaml

```diff
@@ -429,7 +429,7 @@ presubmits:
       preset-k8s-ssh: "true"
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/kubekins-e2e:v20180605-65a672b59-master
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20180625-2232199ea-master
```

Comment:
How is bumping kubekins related to this change? This seem to touch the kubeadm image only

Reply:
kubeadm is always built on top of kubekins, we'll keep both of them up-to-date

Reply:
we have a convenient script to upgrade the e2e images to the current kubetest source etc, usually if we upgrade e2e images we just do that. we *can* manually just bump the kubeadm image if we want to do that though. realistically right now we should be safe to upgrade everything and we want to do that regularly.